### ISSUE TITLE
[BUG] FLAGS : Wool don't recolor when flag is captured

### ISSUE THREAD

**Issue by Oruss7** [slipcor/pvparena#183]
When we click on the flag, we have the wool on head and the flag is white. 
Then we capture it, wool remains white.

![2016-05-03_21 34 32](https://cloud.githubusercontent.com/assets/4788317/14995789/e5e609ce-1176-11e6-8aad-d1688ac372e7.png)

**Comment by slipcor** [comment]
can you check your config? can you pastebin your "blocks" / "spawns" part?

**Comment by Oruss7** [comment]
http://pastebin.com/UY5550ji

**Comment by slipcor** [comment]
@Oruss7 the "flag" part is only the red tip, right? The rest of the flag is white and should stay white all the time, right?

**Comment by Oruss7** [comment]
No, white blocks are flag, the third is no flag because of this issues.

**Comment by Oruss7** [comment]
maybe recolor flag after a player lose the flag ? you have the flag position on the arena config and the team color

**Comment by slipcor** [comment]
So, the right block should be red, as it is the flag block, and the ones to the left to it do not matter, stay red/white?

**Comment by Oruss7** [comment]
The red block is not a flag, others are. There are white because someone pick it and they don't rollback to red color

**Comment by slipcor** [comment]
okay let me rephrase the question. Your config shows only ONE block defined. Can BOTH blocks that are visibly WHITE in the picture be claimed as flag if they are RED before? Or is it only ONE of the two white, and, which one?

**Comment by CCCCarsten** [comment]
I can not confirm. The flag is white when it is stolen. If the flag of the opponents declined again in time, the flag is in the base colored again (blue / red). Everything OK (in my humble opinion).

Kann ich nicht bestätigen. Die Flagge wird weiß, sobald sie geraubt wird. Wird die Flagge dem Gegner rechtzeitig wieder abgenommen, wird die Flagge in der Basis wieder farbig (blau/rot). Alles in Ordnung.

**Comment by Oruss7** [comment]
can you share your arena.config please ?

**Comment by slipcor** [comment]
I still believe it is an issue that is happening because of your specific wool blocks in the picture. either the arena is messing up or I misunderstood you. I really did not understand yet which ones of those (if multiple) should be a flag... most importantly , should it be only one? because if there should be more than one, your arena setup only has values for ONE block.

Thats basically what I am trying to find out and I still have no idea how it should work for you....

**Comment by CCCCarsten** [comment]
Hi @Oruss7,
yes, of course :)). Here is my arenaname.yml:
http://pastebin.com/PyPc22Jj
Good night, Carsten

**Comment by Oruss7** [comment]
Bad coord. Sorry my bad. Working fine.

**Comment by CCCCarsten** [comment]
Hi Oruss7! Would you like to help me ? :))
As a part of discussion you have written to me in the past that you are using command blocks inside a pvp-game. Du you use the command blocks in the lounge, too ? Means: Player joins in the lounge and decide then not to fight and to leave the lounge. Does the player use a command sign or a command block?

**Comment by Oruss7** [comment]
Hi Carten,
Yes i use only commandblock in lounge. I activate `inGameClassSwitch` and i use `sudo @p` 
to emulate player command. 

Examples: 

- `/sudo @p[r=2] pa destruction -j red` to join red team for arena destruction
- `/sudo @p pa -ac protecteur` to switch class to *protecteur* => require inGameClassSwitch
- `/sudo @p[r=2] pa leave` to leave the lounge

**Comment by CCCCarsten** [comment]
Hi Oruss7, thank you very much for your answer. Maybe, that there are one or two questions of me in the next two days. I think you have found a great solution :)

**Comment by CCCCarsten** [comment]
ahhhh.... only a short question ;)): Are you working with lounges? And do you have the situation sometimes that some button oder CommandSign do not work?

**Comment by Oruss7** [comment]
If your lounge is on pvpArena region, make sure you have protection disable. I always make my lounge outside pvpArena region and protect it via my area protect plugin.

**Comment by CCCCarsten** [comment]
@Oruss7 , I would like to look on your server... My minecraft name is ccccarsten, if you are using a whitelist. Is this okay for you?

**Comment by Oruss7** [comment]
Sure, no whitelist. you need to register with `/register ANewpassword repeatPassword` and after `/login password`

**Comment by CCCCarsten** [comment]
Please let me know your IP or your Minecraft server name... :)

**Comment by Oruss7** [comment]
Oh yes xD play.eredrim.fr
